tests/unit/test_charm.py

```diff
@@ -49,13 +50,62 @@ def setUp(self):
         self.addCleanup(self.harness.cleanup)
         self.harness.begin()
 
+    def add_complete_identity_relation(self, harness: Harness) -> None:
+        """Add complete identity-service relation."""
+        credentials_content = {"username": "svcuser1", "password": "svcpass1"}
+        credentials_id = harness.add_model_secret("keystone", credentials_content)
+        app_data = {
+            "admin-domain-id": "admindomid1",
```

Comment:
adding a note for future enhancement to support custom configuration of these values in `charm-microceph`.